src/kontrol/options.py

```diff
@@ -231,14 +231,15 @@ def get_argument_type() -> dict[str, Callable]:
         return FoundryOptions.get_argument_type() | LoggingOptions.get_argument_type() | KOptions.get_argument_type()
 
 
-class LoadStateDiffOptions(LoggingOptions, FoundryOptions):
+class LoadStateOptions(LoggingOptions, FoundryOptions):
```

Comment:
In `test_foundry_prove.py`, there're some imports of classes and functions that got renamed that seem to need adjustment.

Reply:
Addressed!